src/inmanta/util/__init__.py

```diff
@@ -78,6 +78,43 @@ def is_sub_dict(subdct: dict[PrimitiveTypes, PrimitiveTypes], dct: dict[Primitiv
     return not any(True for k, v in subdct.items() if k not in dct or dct[k] != v)
 
 
+def strtobool(val: str) -> bool:
```

Comment:
1. I don't want to even think about the implication of including a piece of licensed code for this little gain
2. This is not correct wrt the behavior of pip https://pip.pypa.io/en/stable/topics/configuration/#environment-variables